### PR TITLE
nautilus-python: fix cross-build

### PR DESCRIPTION
**Pull request by ndowens**

### CODE REVIEW

File: srcpkgs/nautilus-python/template [hunk 1]
```diff
@@ -3,8 +3,11 @@ pkgname=nautilus-python
 version=1.2.3
 revision=3
 build_style=gnu-configure
-configure_args="--enable-gtk-doc PYTHON=python3"
-hostmakedepends="automake libtool pkg-config which"
+configure_args="--enable-gtk-doc PYTHON=python3
+ am_cv_python_pythondir=${XBPS_CROSS_BASE}/${py3_lib}/site-packages
+ am_cv_python_pyexecdir=${XBPS_CROSS_BASE}/${py3_lib}/site-packages
```
Comment (sgn) [inline srcpkgs/nautilus-python/template:8]: This part looks sketchy. `${XBPS_CROSS_BASE}`

I think I'll do this instead.

```diff
diff --git a/srcpkgs/nautilus-python/template b/srcpkgs/nautilus-python/template
index 072aeedb20..2aa4b0f9c6 100644
--- a/srcpkgs/nautilus-python/template
+++ b/srcpkgs/nautilus-python/template
@@ -4,9 +4,8 @@ version=1.2.3
 revision=3
 build_style=gnu-configure
 configure_args="--enable-gtk-doc PYTHON=python3
- am_cv_python_pythondir=${XBPS_CROSS_BASE}/${py3_lib}/site-packages
- am_cv_python_pyexecdir=${XBPS_CROSS_BASE}/${py3_lib}/site-packages
- PYTHON_INCLUDES=-I${XBPS_CROSS_BASE}/${py3_inc}"
+ PYTHON_INCLUDES=-I${XBPS_CROSS_BASE}/${py3_inc}
+ PYTHON_LIBS=-lpython${py3_ver}"
 hostmakedepends="automake libtool pkg-config which python3"
 makedepends="python3-devel python3-gobject-devel gtk-doc nautilus-devel"
 depends="nautilus python3-gobject"
```

File: srcpkgs/nautilus-python/template [hunk 2]
```diff
@@ -13,6 +16,13 @@ license="GPL-2.0-or-later"
...
 
 CFLAGS="-fcommon"
+
+pre_build() {
+	# Needs itself to build HTML_DATA
+	# which fails to be found if cross-built
+	if [ "$CROSS_BUILD" ]; then
+		vsed -i '/DATA/s;$(HTML_DATA);;' docs/Makefile
+	fi
+}
```
Comment (sgn) [inline srcpkgs/nautilus-python/template:28]: Maybe just disable gtkdoc?

File: srcpkgs/nautilus-python/template [hunk 3]
```diff
@@ -3,16 +3,17 @@ pkgname=nautilus-python
 version=1.2.3
 revision=3
 build_style=gnu-configure
-configure_args="--enable-gtk-doc PYTHON=python3"
-hostmakedepends="automake libtool pkg-config which"
-makedepends="python3-devel python3-gobject-devel gtk-doc nautilus-devel"
+configure_args="--disable-gtk-doc PYTHON=python3
```
Comment (sgn) [inline srcpkgs/nautilus-python/template:6]: When the configure_args changed, please bump revision.
@mnabid Do you still want to have gtk-doc? Should we add a build_options instead?

**Comment by mnabid** [comment]
probably add a build_options and disable it by default.
________________________________

**Pull request closed without merging**